docs/pages/application-access/okta/scim-only.mdx

```diff
@@ -0,0 +1,133 @@
+---
+title: Installing SCIM-only Okta integration
+description: How to install a SCIM-only Okta integration
+---
+
+## Prerequisites
+
+(!docs/pages/includes/commercial-prereqs-tabs.mdx!)
+- [Authentication With Okta as an SSO Provider](../../access-controls/sso/okta)
+
+
+## How it works
+User provisioning (and de-provisioning) with SCIM requires two Teleport
+components working together:
+
+ * A SAML Connector that provides SSO login to Teleport for upstream Okta users
+ * A SCIM integration that provisions and de-provisions Teleport user accounts
+   in response to changes in the upstream Okta organization
+
+Both of these Teleport components rely on an Okta SAML Application to act as the
+interface between Teleport and Okta. For consistency, both of
+the Teleport components must use the same Okta Application.
+
+When a user is assigned to the Okta App, either directly or via group
+membership, a corresponding Teleport user account will be created. If the Okta
+user already has a valid temporary Teleport SAML user account (i.e. they have
+logged into the cluster via SAML SSO before SCIM provisioning was enabled), the
+temporary account will automatically be adopted by the SAML integration and
+promoted to a long-lived SCIM-managed account.
+
+<Admonition type="note">
+Currently none of the SCIM user profile traits are stored in the Teleport user, although this may change in future.
+</Admonition>
+
+When a user is unassigned from the Okta App, or is disabled by the Okta Admin, Teleport will immediately delete the user in question, and create a lock that will both immediately terminate any existing sessions and prevent that user from re-using any credentials that have already been issued. This lock will be automatically revoked if the user is subsequently re-provisioned via SCIM, otherwise the lock is permanent and will have to be explicitly deleted to remove.
+
+<Admonition type="warning">
+Okta does not send SCIM updates to Teleport when a user is merely suspended. Even though Okta will prevent a suspended user from logging back into the cluster, any existing sessions will not be terminated, and any pre-issued credentials will be valid for their normal lifetimes.
+</Admonition>
+
+## Installing the Teleport SCIM integration
```

Comment:
The convention in the docs is to use `Step n/d .` for H2 sections in how-to guides. E.g., this would be: `## Step 1/4. Install the Teleport SCIM integration`. We should change the other headings as well.